python/cudf/cudf/tests/test_string.py

```diff
@@ -1828,3 +1828,36 @@ def test_string_str_decode_url(data):
     got = gs.str.url_decode()
     expected = pd.Series([urllib.parse.unquote(url) for url in data])
     assert_eq(expected, got)
+
+
+@pytest.mark.parametrize(
```

Comment:
Better to have two different set of test cases, one which fails for sure and one which passes for sure. Here, there is possibility that, something working might start giving ValueError later and this test case would fail to detect it.